app.rb

```diff
@@ -0,0 +1,45 @@
+@alphabets = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_char(string)
+  @alphabets[string]
+end
+
+def decode_word(word)
+  word.split(/ /).map { |letter| decode_morse letter }.join
+end
+
+def decode_sentence(sentence)
+  sentence.split(/  /).map { |word| decode_word word }.join(' ')
+end
+
```

Comment:
- [ ] `README.md` file: To make your documentation be more descriptive and professional, I think it would be better if you update the both the `## Show your support` and `Getting Started` sections with appropriate information. To help you fill the `Getting Started` section just refer this **[template example](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)**.  
![Screenshot from 2023-04-12 13-08-39](https://user-images.githubusercontent.com/83042742/231531952-5d8775d9-d6bc-47d7-a10f-be0b39400071.png)